src/main/java/com/minecolonies/client/gui/WindowHutFarmer.java

```diff
@@ -229,29 +229,20 @@ public void updateElement(int index, @NotNull Pane rowPane)
 
                 rowPane.findPaneOfTypeByID(TAG_DIRECTION, Label.class).setLabelText(direction);
 
-                if (building.assignFieldManually())
-                {
-                    rowPane.findPaneOfTypeByID(TAG_BUTTON_ASSIGN, Button.class).enable();
-                }
-                else
-                {
-                    rowPane.findPaneOfTypeByID(TAG_BUTTON_ASSIGN, Button.class).disable();
-                }
+                Button assignButton = rowPane.findPaneOfTypeByID(TAG_BUTTON_ASSIGN, Button.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'assignButton' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)